lib/Ebook.php

```diff
@@ -1153,61 +1145,41 @@ public function Validate(): void{
 			$error->Add(new Exceptions\StringTooLongException('Ebook AlternateTitle'));
 		}
 
-		if(isset($this->Description)){
-			$this->Description = trim($this->Description);
-
-			if($this->Description == ''){
-				$error->Add(new Exceptions\EbookDescriptionRequiredException());
-			}
-		}
-		else{
-			$error->Add(new Exceptions\EbookDescriptionRequiredException());
+		$this->Description = trim($this->Description ?? '');
+		if($this->Description == ''){
+			$this->Description = null;
 		}
 
-		if(isset($this->LongDescription)){
-			$this->LongDescription = trim($this->LongDescription);
-
-			if($this->LongDescription == ''){
-				$error->Add(new Exceptions\EbookLongDescriptionRequiredException());
-			}
+		if(isset($this->Description) && strlen($this->Description) > EBOOKS_MAX_STRING_LENGTH){
+			$error->Add(new Exceptions\StringTooLongException('Ebook Description'));
 		}
-		else{
-			$error->Add(new Exceptions\EbookLongDescriptionRequiredException());
-		}
-
-		if(isset($this->Language)){
-			$this->Language = trim($this->Language);
 
-			if($this->Language == ''){
-				$error->Add(new Exceptions\EbookLanguageRequiredException());
-			}
+		$this->LongDescription = trim($this->LongDescription ?? '');
+		if($this->LongDescription == ''){
+			$this->LongDescription = null;
+		}
 
-			if(strlen($this->Language) > 10){
-				$error->Add(new Exceptions\StringTooLongException('Ebook Language: ' . $this->Language));
-			}
+		$this->Language = trim($this->Language ?? '');
+		if($this->Language == ''){
+			$this->Language = null;
 		}
-		else{
-			$error->Add(new Exceptions\EbookLanguageRequiredException());
+
+		if(isset($this->Language) && strlen($this->Language) > 10){
```

Comment:
Make this a nested if, because we later we might want to add more checks if `Language` is not null.

Reply:
Done.